lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -192,7 +192,7 @@ public function getDropIndexSQL($index, $table = null)
 
         return "IF EXISTS (SELECT * FROM sysobjects WHERE name = '$index')
```

Comment:
@zeroedin-bill still this is not an issue of the platform IMO as the platform more or less acts as an independent SQL generator (mostly). The platform does not know about the schema introspection / schema tool. Sounds to me as if this should rather be fixed in the schema manager. Using `IF EXISTS` clauses in the generated SQL is not desired IMO as it silences errors in situations where dropping an index might be expected to fail if it does not exist.
Unfortunately we already have that clause in here which I was not aware of...